docs/development/triage-guide.md

```diff
@@ -0,0 +1,88 @@
+# Triage guide
+
+## What is triage?
+
+Triage is basically filtering the issues/discussions, and categorizing them with the proper labels.
+
+## What a triagist is allowed to do
+
+If you've been given triage rights, you are allowed to do the following things:
+
+- Apply labels to issues
+- Close, reopen, and assign all issues and pull requests
+- Mark duplicate issues and pull requests
+- Request pull request reviews
+- Lock and unlock discussions
+- Individually convert issues to discussions
+
+**Note:** We don't use milestones or project boards.
+
+## Guidelines for triage workflow
+
+The following are guidelines as we cannot cover all situations.
+Use common sense, and just do your best, and you'll do all right.
+Don't be afraid to ask for help.
+
+### Apply labels to issues
+
+All issues should have labels attached to them.
+Read the [issue-labeling guide](https://github.com/renovatebot/renovate/blob/master/docs/development/issue-labeling.md) to get all the necessary info.
+
+In general try to make a good-faith effort to label things correctly.
+
+### Closing issues
+
+You can close a issue yourself if it's:
+
+- Spam
+- Obviously fixed
+
+For really old issues, it's probably a good idea to ask the maintainers to decide if they want to keep or close the issue.
+
+### Closing pull requests
+
+It's not very often that you'll need to close a PR, but you can certainly do it in case of spam or malicious content in the PR diff.
+
+### Reopen issues
+
+Sometimes a bug is fixed with a PR that links to a issue.
+When the PR is merged, the issue is automatically closed.
+Sometimes the bug was not really fixed, and someone says: "Hey this is still broken for me."
+In that case, re-open the issue only if it's definitely the same problem (users often associate different problems together incorrectly).
+Otherwise, ask the user to open a new issue if it seems like it is different.
+
+### Assign issues
+
+You can assign a issue to yourself, so that others know you're going to work on the issue.
+GitHub allows issues to be assigned to any project collaborator or to any non-collaborator who has created or commented on the issue, so you can also assign in either of those cases if it makes sense.
+
+### Mark duplicate issues and pull requests
+
+If you see an issue that's an obvious duplicate, you can mark it as a duplicate and close it.
```

Comment:
Oh if you wanted to search later a label `duplicates` would be way easier than doing a plaintext search on the repository for the word duplicate.
So for that use-case using a label is better than using the GitHub "duplicate of" functionality".

How about this then?

```suggestion
If you see an issue that's an obvious duplicate, attach the `duplicate` label, and close the issue.
```

Reply:
Maybe also mark as duplicate if possible 🙃